torch/distributed/fsdp/_fully_shard/_fully_shard.py

```diff
@@ -177,7 +177,8 @@ def fully_shard(
             which offers parameter/gradient/optimizer state offloading. See
             :class:`OffloadPolicy` and its subclasses for details.
         ignored_params: Optional(Set[nn.Parameter]): The set of parameters that we
-            don't want to shard with FSDP.
+            don't want to shard with FSDP, but will still be moved to the device if
+            necessary.
```

Comment:
If we go with this PR, I think something like this might be helpful:
```
        ignored_params: Optional(Set[nn.Parameter]): The set of parameters
            ignored by FSDP for sharding and gradient reduction; i.e. FSDP will
            treat these parameters in the same way as buffers. Specifically,
            FSDP always determines managed modules by traversing ``module.modules()``
            stopping before nested FSDP submodules. Parameters and buffers
            owned by those managed modules are moved to device if needed during
            FSDP init, but ignored parameters are not sharded nor have gradient
            reduction run for them.
```